tests/main/install-store/task.yaml

```diff
@@ -37,9 +37,5 @@ execute: |
     actual=$(snap install --channel beta --devmode $DEVMODE_SNAP)
     echo "$actual" | grep -Pzq "$expected"
 
-    echo "Install network-manager and do basic smoke test"
```

Comment:
I removed this as it is racy in this PR at least. https://github.com/snapcore/snapd/pull/3825 is a better test